src/Widgets/DataImageWidget/DataImageWidgetComponent.tsx

```diff
@@ -77,22 +75,10 @@ const ImageComponent = connect(function ImageComponent({
 
   const attributeValue = dataItemAttribute.get()
   if (isDataBinary(attributeValue)) {
-    return (
-      <DataBinaryImage
-        dataBinary={attributeValue}
-        alt={widgetAlternativeText}
-        className={className}
-      />
-    )
+    return <DataBinaryImage dataBinary={attributeValue} className={className} />
   }
 
   if (typeof attributeValue !== 'string') return null
 
-  return (
-    <img
-      src={attributeValue}
-      alt={widgetAlternativeText}
-      className={className}
-    />
-  )
+  return <img src={attributeValue} className={className} />
```

Comment:
* [ ] Leaving out `alt` is bad for accessibility. We should set `alt=""`. See https://www.w3.org/WAI/tutorials/images/decision-tree/